common/Contractions.yml

```
@@ -5,21 +5,30 @@ ignorecase: true
 action:
   name: replace
 swap:
-  aren't: are not
-  can't: cannot
-  couldn't: could not
-  didn't: did not
-  don't: do not
-  doesn't: does not
-  hasn't: has not
-  haven't: have not
-  isn't: is not
-  it's: it is
-  shouldn't: should not
-  that's: that is
-  wasn't: was not
-  weren't: were not
-  what's: what is
-  when's: when is
-  where's: where is
-  won't: will not
+  aren['` ]?t: are not
+  can['` ]?t|can not: cannot
+  couldn['` ]?t: could not
+  didn['` ]?t: did not
+  don['` ]?t: do not
+  doesn['` ]?t: does not
+  hasn['` ]?t: has not
+  haven['` ]?t: have not
+  isn['` ]?t: is not
+  it['` ]?s: it is
```

`its`(possessive) will get caught. It's a problem 

```suggestion
  it['` ]s: it is # its would be false negative
```
remove the `?` and add an inline comment

---

The ``its`` possessive isn't desired anyway, SUSE doesn't allow inanimate objects to receive possessives.

---

I agree. The overwhelming majority of the use of ``its`` is undesirable. 

> ```
> To run a new command and start tracing its system calls
> ```

For the sentence above, the existing documentation has 2 relevant rules:
- That we avoid passive language
- That we avoid ascribing possessives to inanimate objects

Both can be satisfied by using ``the relevant``, ``the resulting``, ``the subsequent`` or even ``the ensuing`` instead of ``its`` there. I believe, we should move on with this. Worst case scenario, the rule is easy to update with an example present anyway.

---

IMO, it is fine to use the possessive pronoun like this for inanimate objects in English. I'm not in favor of replacing it with officious-sounding 'resulting' or 'subsequent'. Here's an example: _Neither SUSE LLC, its affiliates, the authors nor the translators shall be held liable for possible errors or the consequences thereof._

---

I see, we often use ``the relevant`` in cases where I've seen it. ``subsequent`` is a phonetically lovely word, but definitely not a great fit for documentation.